Formula/s/slepc.rb

```diff
@@ -0,0 +1,48 @@
+class Slepc < Formula
+  desc "Scalable Library for Eigenvalue Problem Computations (real)"
+  homepage "https://slepc.upv.es"
+  url "https://slepc.upv.es/download/distrib/slepc-3.22.2.tar.gz"
+  sha256 "b60e58b2fa5eb7db05ce5e3a585811b43b1cc7cf89c32266e37b05f0cefd8899"
+  license "BSD-2-Clause"
+
+  livecheck do
+    url "https://slepc.upv.es/download/distrib/"
+    regex(/href=.*?slepc[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "open-mpi"
+  depends_on "openblas"
+  depends_on "petsc"
+
+  uses_from_macos "python" => :build
+
+  conflicts_with "slepc-complex", because: "slepc must be installed with either real or complex support, not both"
+
+  def install
+    ENV["PETSC_DIR"] = Formula["petsc"].prefix.realpath
+    ENV["SLEPC_DIR"] = buildpath
+
+    # This is not an autoconf script so cannot use `std_configure_args`
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "all"
+    system "make", "install", "PYTHON=#{which("python3")}"
+
+    # Avoid references to Homebrew shims
+    rm(lib/"slepc/conf/configure-hash")
+  end
+
+  test do
+    pform = "petsc"
+    flags = %W[-I#{include} -L#{lib} -lslepc -I#{Formula[pform].include} -L#{Formula[pform].lib} -lpetsc]
+    flags << "-Wl,-rpath,#{lib} -Wl,-rpath,#{Formula[pform].lib}" if OS.linux?
+    system "mpicc", pkgshare/"examples/src/eps/tutorials/ex2.c", "-o", "test", *flags
+    cmd = ""
+    cmd << "LD_LIBRARY_PATH=#{lib} " if OS.linux?
+    cmd << "./test -terse"
+    output = shell_output(cmd)
```

Comment:
I 
```suggestion
    ENV["LD_LIBRARY_PATH"] = lib.to_s if OS.linux?
    output = shell_output("./test -terse")
```
And same for the other variant.

Reply:
Changed